Makefile

```diff
@@ -144,9 +146,13 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+GOLANGCI_TMP_FILE = .golangci.mktmp.yml
 .PHONY: fmt
-fmt: ## Run go fmt against code.
+fmt: golangci-lint yq ## Run go fmt against code.
```

Comment:
Probably the help line should be now more abstract from `go fmt` since now it runs more than that. Like "Format source code"

Reply:
Right! I knew I forgot about this :) Thanks

Reply:
Fixed in 6e0268b